Platforms/QemuQ35Pkg/Plugins/QemuRunner/QemuRunner.py

```diff
@@ -78,7 +78,12 @@ def Runner(env):
         else:
             logging.critical("Virtual Drive Path Invalid")
 
-        args += " -machine q35,smm=on" #,accel=(tcg|kvm)"
+        if env.GetBuildValue("SMM_ENABLED") is None or env.GetBuildValue("SMM_ENABLED").lower() == "true":
+            smm_enabled = "on"
+        else:
+            smm_enabled = "off"
+
+        args += " -machine q35,smm=" + smm_enabled #,accel=(tcg|kvm)"

```

Comment:
When SMM is not enabled, do we want to enable the accelerator by default, just for the better performance?

Reply:
I don't think so. because we don't know do sure if it will be available. From what I have heard a lot of QEMU build dont properly support hypervisor acceleration, so it's probably better left as a flag. I should do that though.